README.md

```diff
@@ -1,15 +1,49 @@
-# song-stitch
+# Song Stitch
 
 <p align="center">
   <img alt="SongStitch Logo" src="https://raw.githubusercontent.com/SongStitch/song-stitch/main/public/logo.png" width="500px"/>
 </p>
 
 <div align="center">
 
-[![Website Status](https://img.shields.io/website?style=flat-square&up_message=UP&url=https%3A%2F%2Fsongstitch.art%2F
-)](https://songstitch.art/)
+[![Website Status](https://img.shields.io/website?style=flat-square&up_message=UP&url=https%3A%2F%2Fsongstitch.art%2F)](https://songstitch.art/)
 [![CI status](https://img.shields.io/github/actions/workflow/status/SongStitch/song-stitch/deploy.yml?branch=main&style=flat-square)](https://github.com/SongStitch/song-stitch/actions?query=branch%3Amain)
 [![License](https://img.shields.io/github/license/SongStitch/song-stitch?style=flat-square)](/LICENCE)
 
 </div>
 
+<div align="center">
+A <em>blazingly fast</em> web application for generating LastFM collages, written in Go.
+</div>
+
+<br/>
+
+<p align="center">
+  <img alt="SongStitch Logo" src="https://raw.githubusercontent.com/SongStitch/song-stitch/main/collage.png" width="300px"/>
```

Comment:
This image can't be seen? https://github.com/SongStitch/song-stitch/blob/802d0e30a30a776b9197672693c291ddb7703a0d/README.md

I think once it's in main, you can link to it directly. This is useful if the readme is viewed outside the repo. We can commit the image in `main` (maybe under a `docs/`  folder or just in the root dir), then update the relative link to the hotlink

Would also be good if you run [imageoptim](https://imageoptim.com/mac) on the image with lossless compression + strip metadat

Reply:
Yeah it won't be there until this is merged. Tbh I wasn't sure the best way to do that.